### PR TITLE
Reenable dep and revert workaround for github connectivity issues

### PR DESCRIPTION
**Pull request by krdln**
Linked PR in docker driver: https://github.com/codilime/contrail-windows-docker/pull/113

### CODE REVIEW

**Comment by krdln** [comment]
Note: Building with `DRIVER_BRANCH` manually set to `reenable-dep`

File: CIScripts/Build/BuildFunctions.ps1 [hunk 1]
```diff
@@ -137,8 +137,27 @@ function Invoke-DockerDriverBuild {
     })
 
     $Job.Step("Intalling MSI builder", {
+        # go-msi needs specific version of its dependencies,
```
Comment (kaweue) [inline CIScripts/Build/BuildFunctions.ps1]: We think it would be better to use MSIs to install go-msi, dep and ginkgo on builder in ansible scripts.

Comment (kaweue) [reply CIScripts/Build/BuildFunctions.ps1]: For ginkgo you can:
1) Prebuild it and install it.
2) Do nothing.

**Review by kaweue — CHANGES_REQUESTED** [review]

File: CIScripts/Build/BuildFunctions.ps1 (hunk 1 above)
Comment (krdln) [reply CIScripts/Build/BuildFunctions.ps1]: Moved this change to https://github.com/codilime/contrail-windows-ci/pull/13

I'm going to remove this commit and rebase.

**Review by kaweue — APPROVED** [review]

**Review by katrybacka — APPROVED** [review]

**Pull request merged**